creator-node/src/routes/files.js

```diff
@@ -474,37 +355,59 @@ const getCID = async (req, res) => {
     if (!found) {
       throw new Error('Not found in network')
     }
-    decisionTree.push({
-      stage: `FIND_CID_IN_NETWORK_COMPLETE`,
-      time: `${Date.now() - startMs}ms`
-    })
+  } catch (e) {
+    prometheusResult.mode = 'abort_cid_not_found_in_network'
+    req.logger.error(
+      { cid: CID },
+      `Could not find cid in network: ${e.message}`
+    )
+    return sendResponseWithMetric(
+      req,
+      res,
+      errorResponseServerError('CID not found in network'),
+      prometheusResult
+    )
+  }
 
-    startMs = Date.now()
```

Comment:
we lose some information with the timings of each individual stage inside this but i think thats probably ok

Reply:
yeah i think the metrics will give us insight on durations. and we can always add back the timings if we feel the need for them